.circleci/config.yml

```diff
@@ -78,8 +78,9 @@ jobs:
             export PATH="/tmp/go/bin:$PATH"
             export GOPATH=/tmp/go
             cd /tmp/go/src/github.com/FactomProject/factomd/
-            go test $(glide nv | grep -v Utilities | grep -v longTest | grep -v peerTest | grep -v simTest) -vet=off -v -cover -coverprofile=coverage.out
+            go test $(glide nv | grep -v Utilities | grep -v longTest | grep -v peerTest | grep -v simTest | grep '...') -vet=off -v -cover -coverprofile=coverage.out
```

Comment:
Is the `grep ...` supposed to grep the regex of any 3 character sequence? Or grep for `grep "\.\.\."`

Reply:
it looks for three literal '...' - one of the packages returned from `glide -nv` is "."